server/db/schema/index.ts

```diff
@@ -20,3 +20,4 @@ export * from './sponsored-research-projects.schema';
 export * from './staff.schema';
 export * from './student-academic-details.schema';
 export * from './students.schema';
+export * from './events.schema';
```

Comment:
This file is also alphabetically ordered. And in light of the previous comment, this would be `club-events.schema`